### ISSUE TITLE
Option to start-all and start-group/start-tag

### ISSUE THREAD

**Issue by kwill** [albertodonato/sshoot#3]
**User story:** I am a worker requiring sshuttle access to a set of servers. I want to connect to all of them at once, so that I save time and don't forget one.

**Suggested solutions:**
* Add a `sshoot start all` option, which starts all registered connections.
* Add a grouping or tagging option, which assigns each connection to a group, or assigns tags to a connection. Then add a `sshoot start group` or `sshoot start tag` option, which starts all connections in a group, or starts all connections with a tag.

**Workaround:**
* Maintain a script or scripts that call `sshoot start X, sshoot start Y, etc`

**Comment by albertodonato** [comment]
Thaks for the suggestion. Rather than adding an option for a specific use case, I'd rather extend the `list` command to support multiple formas (like CSV and JSON, for instance), and column selection.
At that point, one could just do something like:

```
sshoot list --format csv --columns name | xargs sshoot start 
```

This would work for stop and possibly other commands too

**Comment by albertodonato** [comment]
I've added support for listing profiles in other formats in addition to table (JSON, YAML, CSV).
As suggested, profile names can be extracted easily in one of the following ways, to start/stop them:

```
sshoot list -f csv | sed 1d | cut -d, -f1 | xargs -n1 sshoot start
```
or
```
sshoot list -f json | jq -r 'keys[]' | xargs -n1 sshoot start
```

**Comment by albertodonato** [comment]
Fixed in 1.3.0
